.travis.yml

```diff
@@ -81,10 +81,10 @@ matrix:
         - BUILD_TYPE=sbt
         - PROJECT=reindexer
     - env:
-        - BUILD_TYPE=python
+        - TASK=lint-lambdas
```

Comment:
These were intentionally generic, I think the BUILD_TYPE should be a more general case rather than so closely linked to the project.